robottelo/config/facade.py

```diff
@@ -284,6 +353,21 @@ def _dispatch_computed_value(self, key):
         self._add_to_cache(key, value)
         return value
 
+    def _dispatch_robottelo_value(self, key):
```

Comment:
Maybe I'm missing something obvious here, but its not clear to me why this method is actually needed given the changes you've made?  I'm not seeing what the existing use of `_dispatch_computed_value` or `_get_from_configs` would miss that this is handling.

At a minimum include a docblock here explaining the necessity.

Reply:
@mshriver I have listed the reason of this description on the Description of this PR. Still copying here :
```
New Robottelo dispatch value func which returns data using settings object for DynaConf as `setings.screenshots_path` instead of settings.robottelo object as `settings.robottelo.screenshots_path`. This notation is being used by all the tests in robottelo, and hence we need a big refactor for the tests/helpers using settings from robottelo. So this saves from big refactoring.
```

Reply:
I didnt include docblock as none of other funcs have it :)

Reply:
That's not a good pattern to follow, and violates our code standards, which dictate all classes, methods, and functions will have docstrings.

Reply:
Thanks for explaining the need here.

I think ultimately its a pattern that _should_ be refactored throughout the framework. Access to settings values should be explicit - if I'm working on a test and referencing `settings.screenshot_path` instead of `settings.robottelo.screenshot_path`, I've lost the context of where my settings are defined. This layer obfuscates that for very minimal convenience.

Due to the incremental transition to using dynaconf, this refactor doesn't have to occur in this PR.

I think the docblock for this method should include a TODO to refactor `settings` references to `settings.robottelo` where appropriate for the new config.  Its a very easy find+replace task.

Reply:
Sure! and Make Sense, since the PR is raised as a workaround for unblocking the DynaConf CI, my plan is to point the settings to settings.robottelo in a separate PR where we will also remove the INIReader reference altogether, even we will remove the facade implementation itself in that PR, so the need of this will be auto removed.